qcfractal/interface/client.py

```diff
@@ -66,17 +65,19 @@ def __str__(self):
         ret += "username='{}')".format(self.username)
         return ret
 
-    def _request(self, method, service, payload):
+    def _request(self, method, service, payload, noraise=False):
```

Comment:
Specially an error is thrown if the response is not 200. For the manager itself, we want the exact response (including non-200's) so that we can deal with it from there.